backend-rust/src/graphql_api/stable_coin.rs

```diff
@@ -0,0 +1,291 @@
+use async_graphql::{Context, Object, SimpleObject};
+use chrono::{Duration, Utc};
+use serde::Deserialize;
+use std::{
+    collections::{HashMap, HashSet},
+    fs::File,
+    io::BufReader,
+};
+
+#[derive(Debug, Clone, Deserialize, SimpleObject)]
+pub struct StableCoin {
+    name:                String,
+    symbol:              String,
+    decimal:             u8,
+    total_supply:        i64,
+    circulating_supply:  i64,
+    value_in_doller:     f64,
+    total_unique_holder: Option<i64>,
+    transfers:           Option<Vec<Transfer>>, // Transfers sorted by date
+    holding:             Option<Vec<Holding>>,
```

Comment:
```suggestion
    holdings:             Option<Vec<Holding>>,
```

Reply:
fixed : https://github.com/Concordium/concordium-scan/blob/629b4ec195a58d0f24ac7cc7d4727a9ed584725d/backend-rust/src/graphql_api/stable_coin.rs#L26

Please review and resolve the comment